### PR TITLE
Added shortcuts ctrl+F4 -> close current tab, ctrl+F5 -> reload tab, ctr...

### PR DESCRIPTION
**Pull request by khandelwalYash**
...l+F11 -> toggle between normal and fullscreen

### CODE REVIEW

File: webactivity.py [hunk 1]
```diff
@@ -493,8 +495,8 @@ def _alert_cancel_cb(self, alert, response_id):
         self.remove_alert(alert)
 
     def _key_press_cb(self, widget, event):
-        key_name = Gdk.keyval_name(event.keyval)
-        browser = self._tabbed_view.props.current_browser
+        key_name = Gdk.keyval_name(event.keyval)    
+        browser = self._tabbed_view.props.current_browser
```
Comment (SAMdroid-apps) [inline webactivity.py]: Please don't add extra spaces!

File: webactivity.py [hunk 2]
```diff
@@ -522,19 +524,28 @@ def _key_press_cb(self, widget, event):
...
                 _logger.debug('keyboard: close tab')
                 self._tabbed_view.close_tab()
+            elif key_name ==  'F11':
+                if self.count % 2 == 0 :   
+                    self._primary_toolbar.hide()
+                    self.count = self.count + 1
+                else:
+                    self._primary_toolbar.show_all()
+                    self.count = self.count + 1
+
```
Comment (SAMdroid-apps) [inline webactivity.py]: Can we just the built in fullscreen function?  I think the ui should keep the little "exit fullscreen" icon that you get when you click the button.

Also (CC @godiard, @tchx84), maybe this is better suited in the toolkit?

File: webactivity.py [hunk 3]
```diff
@@ -522,19 +524,28 @@ def _key_press_cb(self, widget, event):
...
+                else:
+                    self._primary_toolbar.show_all()
+                    self.count = self.count + 1
+                
             else:
                 return False
 
             return True
 
+
```
Comment (SAMdroid-apps) [inline webactivity.py]: :angry: No more lines plz!  We don't want to waste bytes!  (Also it isn't with the style)

File: webactivity.py (hunk 1 above)
Comment (khandelwalYash) [inline webactivity.py]: ok my mistake...this will not happen from next time :100:

File: webactivity.py (hunk 2 above)
Comment (khandelwalYash) [inline webactivity.py]: yeah this could be done :)

**Comment by davelab6** [comment]
@khandelwalYash wow its been a while.... would you be willing to work on this further? :)

**Comment by i5o** [comment]
Hello?

btw, why F4, shouldn't it be Ctrl W? (I have not used browse for a while, maybe that's already implemented.)

**Comment by quozl** [comment]
From memory ctrl+f4 is in an ICCM standard for closing windows, so that's one reason to add it.

Browse already supports ctrl+w, implemented within WebKit itself.

Perhaps ctrl+f4 should be pushed upstream to WebKit rather than implemented in Browse.

**Comment by quozl** [comment]
Needs merge conflict resolution, please re-open when resolved.

**Pull request closed without merging**